lib/usdUfe/ufe/UsdClipboard.h

```diff
@@ -0,0 +1,92 @@
+#ifndef USDCLIPBOARD_H
+#define USDCLIPBOARD_H
+
+//
+// Copyright 2024 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include <usdUfe/base/api.h>
+
+#include <pxr/usd/usd/common.h>
+#include <pxr/usd/usdUtils/stageCache.h>
+
+namespace USDUFE_NS_DEF {
+
+//! \brief Class to handle clipboard USD data.
+class USDUFE_PUBLIC UsdClipboard
+{
+public:
+    // Clipboard file name and format.
+    static constexpr auto clipboardFileName = "UsdUfeClipboard.usd";
```

Comment:
Default clipboard name from UsdUfe. Can be overridden, such as by MayaUsd and/or LookdevX.